verksamhetsomraden/verksamhetsomraden.html

```diff
@@ -0,0 +1,368 @@
+                                href="https://www.myh.se/Om-webbplatsen/Anvanda-webbplatsen"
+                                >Använda webbplatsen</a
+                              >
+                            </li>
+                            <li>
+                              <a
+                                href="https://www.myh.se/Om-webbplatsen/Webbkakor-pa-den-har-webbplatsen"
+                                >Om webbkakor (cookies)</a
+                              >
+                            </li>
+                            <li><a href="https://www.myh.se/Webbkarta">Webbkarta</a></li>
+                            <li>
+                              <a
+                                href="https://www.myh.se/Om-webbplatsen/Tillganglighetsredogorelse-"
+                                >Tillgänglighetsredogörelse</a
+                              >
+                            </li>
+                          </ul>
+                        </div>
+                      </div>
+                    </div>
+                    <div class="footer__item">
+                      <div class="footer__links">
+                        <a href="#" id="address-link" class="footer__link"
+                          >Myndigheten för yrkeshögskolan</a
+                        >
+                        <a href="#" id="youtube-link" class="footer__link">Youtube</a>
+                      </div>
+                      <div class="footer__card">
+                        <div
+                          id="address-info"
+                          class="footer__card-info footer__card-info--active"
+                        >
+                          <h4>Adress</h4>
+                          <p>
+                            Myndigheten för yrkeshögskolan
+                            <br />
+                            Box 145
+                            <br />
+                            721 05 Västerås
+                          </p>
+                        </div>
+                        <div id="youtube-info" class="footer__card-info">
+                          <h4>Youtube</h4>
+                          <a
+                            href="https://www.youtube.com/channel/UC8iPn0eKy-5vQ6Mx_-ep-KA"
+                          >
+                            MYH Youtube-kanal
+                          </a>
+                        </div>
+                      </div>
+                    </div>
+                  </div>
+          
+                  <div class="footer__publisher">
+                    <h4>Ansvarig utgivare</h4>
+                    <p>Thomas Persson, generaldirektör MYH</p>
+                  </div>
+                </div>
+              </footer>
+           
+              <script
+                src="https://kit.fontawesome.com/9e640bf9f4.js"
+                crossorigin="anonymous"
+              ></script>
+
+        </div>
+
+
+<script src="footer.js"></script>
+<script src="main.js"></script>
```

Comment:
This gives javascript error in verksamhetsomraden-page because there's no carousel slider in verksamhet. Should be safe to remove ```main.js``` script as no other functionality depends on it.